workflows/files/ZWECONF.xml

```diff
@@ -2118,51 +2033,6 @@ echo '            enabled: false' >> "${instance-zowe_runtimeDirectory}/zowe.yam
-echo '          # certificate:' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
-echo '          #   keystore:' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
-echo '          #     alias: ""' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
-#end
-echo '' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
-echo '    # If we customize this to use different external certificate, than should also' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
-echo '    # define "server.internal.ssl.certificate" and enable "server.internal.ssl.enabled".' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
-echo '    # certificate:' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
-echo '    #   keystore:' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
-echo '    #     alias: ""' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
-echo '' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
-echo '  # >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
-#if (${instance-components_metrics_service_enabled} == "true" )
-echo '  metrics-service:' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
-echo '    enabled: ${instance-components_metrics_service_enabled}' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
-echo '    port: ${instance-components_metrics_service_port}' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
-echo '    debug: ${instance-components_metrics_service_debug}' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
-#end
-#if (${instance-components_metrics_service_enabled} == "false" )
-echo '  metrics-service:' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
-echo '    enabled: ${instance-components_metrics_service_enabled}' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
-echo '    port: 7551' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
-echo '    debug: false' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
-#end
-echo '' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
-echo '  # >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
-#if (${instance-components_cloud_gateway_enabled} == "true" )
-echo '  cloud-gateway:' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
-echo '    enabled: ${instance-components_cloud_gateway_enabled}' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
-echo '    port: ${instance-components_cloud_gateway_port}' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
-echo '    debug: ${instance-components_cloud_gatewaye_debug}' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
-#end
-#if (${instance-components_cloud_gateway_enabled} == "false" )
-echo '  cloud-gateway:' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
-echo '    enabled: ${instance-components_cloud_gateway_enabled}' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
-echo '    port: 7563' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
-echo '    debug: false' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
 #end
```

Comment:
line 2087, there is issue, that velocity macro is used with condition components_cloud_gateway_enabled == "false", but there is step for variable set components_caching_service_storage_infinispan_jgroups_port, which condition is based on components_caching_service_storage_mode} == "infinispan". Because of this, step gets skipped, and variable is never initialized, which leads to this in end workflow. Add velocity condition before **echo '          port: ${instance-components_caching_service_storage_infinispan_jgroups_port}** where there will be test for components_caching_service_storage_mode, and based on that you would either inject instance-components_caching_service_storage_infinispan_jgroups_port or port: 7600 as is done for components_caching_service_enabled} == "false" 
![image](https://github.com/user-attachments/assets/027c6f96-c6db-4eae-b577-5a16d60ec370)